ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -88,7 +88,7 @@
     "Name": "Add case number",
     "Description": "Add case number",
     "DisplayOrder": 5,
-    "PreConditionState(s)": "Submitted",
+    "PreConditionState(s)": "Submitted;PREPARE_FOR_HEARING",
```

Comment:
this event triggers call to robotics, is that OK in prepare for hearing state?

Reply:
It doesn't send if the number was already there.